charts/vaultwarden/templates/ingress.yaml

```diff
@@ -31,8 +31,10 @@ metadata:
     {{- end }}
     nginx.ingress.kubernetes.io/connection-proxy-header: "keep-alive"
     nginx.ingress.kubernetes.io/enable-cors: "true"
+    {{- if $ingress.tls }}
     nginx.ingress.kubernetes.io/ssl-redirect: "true"
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    {{ - end }}
```

Comment:
Please fix this, there should be no space before `-`. Thanks.

Reply:
Are you able to use github's suggestion feature, and I can commit that? I am away from my development machine at the moment